h2o-core/src/test/java/hex/tfidf/DocumentFrequencyTest.java

```diff
@@ -0,0 +1,153 @@
+package hex.tfidf;
+
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.TestUtil;
+import water.fvec.Frame;
+import water.fvec.TestFrameBuilder;
+import water.fvec.Vec;
+import water.util.FileUtils;
+
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.zip.GZIPInputStream;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+
+public class DocumentFrequencyTest extends TestUtil {
+
+    private static final String BIGDATA_TESTS_ENV_VAR_NAME = "testDFBigdata";
+    
+    @BeforeClass()
+    public static void setup() { stall_till_cloudsize(1); }
+
+    @Test
+    public void testDocumentFrequenciesSmallData() {
+        Frame fr = getSimpleTestFrame();
+
+        Map<String, Long> expectedDocumentFrequencies = new HashMap() {{
+            put("A", 2L);
+            put("B", 2L);
+            put("C", 2L);
+            put("Z", 1L);
+        }};
+        int expectedDfValuesCnt = expectedDocumentFrequencies.size();
+
+        try {
+            DocumentFrequency df = new DocumentFrequency();
+            Frame outputFrame = df.compute(fr);
+            long outputRowsCnt = outputFrame.numRows();
+
+            Assert.assertEquals(expectedDfValuesCnt, outputRowsCnt);
```

Comment:
Try to add messages to all asserts, please. :)

Reply:
👍 I added messages to all asserts.